src/card/index.js

```diff
@@ -59,14 +58,6 @@ const Card = ({
   );
 };
 
-Card.defaultProps = {
-  renderImage: true,
-  imagePosition: "left",
-  data: null,
-  classNamePrefix: null,
-  cardOption: null,
-};
```

Comment:
defaultProps is deprecated by React.

https://github.com/facebook/react/pull/16210